[no issue] content delivery arch (#937)
aziontech/docs: src/content/docs/pt-br/pages/arquitetura/content-delivery/content-delivery.mdx

---

> title: Aumente a velocidade e confiabilidade de content delivery no edge

Reviewer: ```suggestion
title: Aumente a velocidade e confiabilidade da entrega de conteúdo no edge
```

description: Use conteúdo nativo do edge para substituir CDNs legadas e aproveitar a velocidade e confiabilidade do edge, reduzindo custos.
meta_tags: edge, aplicações, aplicações web, api, cache, cdn
namespace: docs_arch_content_delivery
permalink: /documentacao/arquitetura/edge-application/content-delivery/
---

> Arquiteturas de content delivery nativas do edge permitem que os desenvolvedores substituam CDNs legadas e aproveitem a velocidade e confiabilidade da edge network, reduzindo custos. A Azion minimiza a latência e permite conteúdo dinâmico e recursos avançados, processando requisições no edge mais próxima dos usuários finais. Esta abordagem abrangente permite aos desenvolvedores construir e implantar aplicações de alto desempenho e escaláveis que melhoram a experiência do usuário e melhoram as pontuações do Core Web Vitals, superando as limitações das soluções tradicionais de CDN.

Reviewer: ```suggestion
Arquiteturas de content delivery nativas do edge permitem que os desenvolvedores substituam CDNs legadas e aproveitem a velocidade e confiabilidade da edge network, reduzindo custos. A Azion minimiza a latência e permite conteúdo dinâmico e recursos avançados, processando requisições no edge mais próximo dos usuários finais. Esta abordagem abrangente permite aos desenvolvedores construir e implantar aplicações de alto desempenho e escaláveis que melhoram a experiência do usuário e melhoram as pontuações do Core Web Vitals, superando as limitações das soluções tradicionais de CDN.
```

> Arquiteturas de content delivery nativas do edge permitem que os desenvolvedores substituam CDNs legadas e aproveitem a velocidade e confiabilidade da edge network, reduzindo custos. A Azion minimiza a latência e permite conteúdo dinâmico e recursos avançados, processando requisições no edge mais próximo dos usuários finais. Esta abordagem abrangente permite aos desenvolvedores construir e implantar aplicações de alto desempenho e escaláveis que melhoram a experiência do usuário e melhoram as pontuações do Core Web Vitals, superando as limitações das soluções tradicionais de CDN.

Author: ```suggestion
As Arquiteturas de entrega de conteúdo nativas do edge permitem que os desenvolvedores substituam CDNs legadas e aproveitem a velocidade e confiabilidade da edge network, reduzindo custos. A Azion minimiza a latência e permite conteúdo dinâmico e recursos avançados, processando requisições no edge mais próximo dos usuários finais. Esta abordagem abrangente permite aos desenvolvedores construir e implantar aplicações de alto desempenho e escaláveis que melhoram a experiência do usuário e melhoram as pontuações do Core Web Vitals, superando as limitações das soluções tradicionais de CDN.
```

Esta solução é ideal para organizações que procuram modernizar sua estratégia de content delivery, reduzir os custos de infraestrutura e oferecer experiências digitais excepcionais aos seus usuários.

---

## Diagrama da arquitetura de content delivery

> ![Visão geral de uma requisição/resposta através da Edge Application](/assets/docs/images/uploads/content-delivery-arch.png)

Reviewer: ```suggestion
![Visão geral de uma requisição/resposta através do Edge Application](/assets/docs/images/uploads/content-delivery-arch.png)
```

### Fluxo de dados de content delivery

1. Um cliente envia uma requisição HTTP ou HTTPS para um domínio associado a uma edge application.

> 2. No edge node, o Rules Engine processa a requisição, permitindo políticas de cache e comportamento de otimização de imagem durante a fase de requisição.

Reviewer: ```suggestion
2. No edge node, o Rules Engine processa a requisição, permitindo políticas de cache e comportamento de otimização de imagem durante a fase de request.
```

3. A requisição é então avaliada pela camada de edge cache. Se uma chave de cache existir para o objeto solicitado, uma correspondência é encontrada e o objeto é entregue a partir do cache.

> 4. Se configurado, a Otimização de Imagem recupera a imagem da Origem e executa algoritmos de processamento de imagem. O edge node armazena a imagem otimizada no cache.

Author: ```suggestion
4. Se configurado, o Image Optimization recupera a imagem da Origem e executa algoritmos de processamento de imagem. O edge node armazena a imagem otimizada no cache.
```

> 4. Se configurado, a Otimização de Imagem recupera a imagem da Origem e executa algoritmos de processamento de imagem. O edge node armazena a imagem otimizada no cache.

Reviewer: ídem

Author: creio que nao

5. Se uma chave de cache não existir para o objeto, a requisição é encaminhada para o servidor de origem.
6. O servidor de origem responde com o conteúdo, que é então armazenado no cache no edge node. Uma chave de cache para o objeto é gerada e incluída no cabeçalho HTTP X-Cache-Key da resposta.
7. Antes do objeto ser retornado ao cliente, quaisquer políticas e diretivas adicionais aplicáveis ao objeto são processadas e aplicadas pelo Rules Engine.
8. Finalmente, o conteúdo é entregue ao usuário.

---

## Componentes

> - [Edge Application](/pt-br/documentacao/produtos/build/edge-application/): configure uma edge application para configurar políticas de entrega e cache, criar regras para automatizar atribuições de políticas de cache com base no tipo de conteúdo e determinar como o conteúdo é armazenado em cache.

Reviewer: ```suggestion
- [Edge Application](/pt-br/documentacao/produtos/build/edge-application/): crie uma edge application para configurar políticas de entrega e cache, criar regras para automatizar atribuições de políticas de cache com base no tipo de conteúdo e determinar como o conteúdo é armazenado em cache.
```

>     - [Edge Cache](/pt-br/documentacao/produtos/build/edge-application/edge-cache/): complemento global para armazenar conteúdo no edge.

Reviewer: ```suggestion
    - [Edge Cache](/pt-br/documentacao/produtos/build/edge-application/edge-cache/): add-on global para armazenar conteúdo no edge.
```

    - [Application Accelerator](/pt-br/documentacao/produtos/build/edge-application/application-accelerator/): usado para personalizar regras de otimização de cache e chaves e cookies de cache com base em padrões estabelecidos.
    - [Rules Engine](/pt-br/documentacao/produtos/build/edge-application/rules-engine/): uma ferramenta para configurar os cenários em que uma política de cache específica é aplicada.
    - [Image Processor](/pt-br/documentacao/produtos/build/edge-application/image-processor/): que permite a manipulação de imagens através da linha de requisição.

---

## Implementação

1. Crie uma edge application: você deve primeiro criar a camada de aplicação de content delivery no edge da rede.

>     - No Console, crie uma aplicação usando o botão **+ Criar** na página inicial.

Reviewer: ```suggestion
    - No Console, crie uma aplicação usando o botão **+ Create** na página inicial.
```

    - Use a API da Azion para criar uma edge application.

>     - Use a edge application.
>     - Use a API da Azion para criar uma edge application.

Reviewer: ```suggestion
    - Use a CLI da Azion para criar e implementar uma edge application.
    - Selecione o template Dynamic and Static File Optimization.
```

5. Configure o Image Processor: permita a manipulação de imagens através da linha de requisição.
    - No Console, vá para a página de configurações da sua aplicação e adicione o Image Processor.
    - Use a API da Azion para configurar o Image Processor.
4. Aplique políticas de cache em cenários específicos: use o Rules Engine para aplicar políticas específicas com base em várias condições. Crie duas regras:
    - Arquivos estáticos: para arquivos estáticos como música, vídeo ou executáveis, aplique a política de cache apropriada.
    - Imagens: para arquivos de imagem, aplique a política de cache apropriada.
5. Aponte o domínio para Azion: atualize suas configurações de DNS para direcionar o registro CNAME do seu domínio para o endpoint Azion fornecido para o seu domínio.
6. Teste e monitore: após configurar a entrega do seu conteúdo, teste-a rigorosamente para garantir que está funcionando conforme esperado e monitore os acessos recebidos.
7. Faça ajustes nas políticas de cache, regras ou outras configurações conforme necessário, com base nas métricas de desempenho e feedback dos usuários.

---

## Documentação relacionada

> - [Application Accelerator](https://www.azion.com/en/documentation/products/build/edge-application/application-accelerator/).

Reviewer: ```suggestion
- [Application Accelerator](/pt-br/documentacao/produtos/build/edge-application/application-accelerator/).
```

---
title: Aumente a velocidade e confiabilidade de content delivery no edge
description: Use conteúdo nativo do edge para substituir CDNs legadas e aproveitar a velocidade e confiabilidade do edge, reduzindo custos.
meta_tags: edge, aplicações, aplicações web, api, cache, cdn
namespace: docs_arch_content_delivery
permalink: /documentacao/arquitetura/edge-application/content-delivery/
---

> As Arquiteturas de entrega de conteúdo nativas do edge permitem que os desenvolvedores substituam CDNs legadas e aproveitem a velocidade e confiabilidade da edge network, reduzindo custos. A Azion minimiza a latência e permite conteúdo dinâmico e recursos avançados, processando requisições no edge mais próximo dos usuários finais. Esta abordagem abrangente permite aos desenvolvedores construir e implantar aplicações de alto desempenho e escaláveis que melhoram a experiência do usuário e melhoram as pontuações do Core Web Vitals, superando as limitações das soluções tradicionais de CDN.

Reviewer: ```suggestion
As [Arquiteturas de entrega de conteúdo nativas do edge](https://www.azion.com/pt-br/solucoes/distribuicao-de-conteudo/) permitem que os desenvolvedores substituam CDNs legadas e aproveitem a velocidade e confiabilidade da edge network, reduzindo custos. A Azion minimiza a latência e permite conteúdo dinâmico e recursos avançados, processando requisições no edge mais próximo dos usuários finais. Esta abordagem abrangente permite aos desenvolvedores construir e implantar aplicações de alto desempenho e escaláveis que melhoram a experiência do usuário e melhoram as pontuações do Core Web Vitals, superando as limitações das soluções tradicionais de CDN.
```

    - Use a API da Azion para configurar o Rules Engine.
5. Configure o Image Processor: permita a manipulação de imagens através da linha de requisição.
    - No Console, vá para a página de configurações da sua aplicação e adicione o Image Processor.
    - Use a API da Azion para configurar o Image Processor.
4. Aplique políticas de cache em cenários específicos: use o Rules Engine para aplicar políticas específicas com base em várias condições. Crie duas regras:
    - Arquivos estáticos: para arquivos estáticos como música, vídeo ou executáveis, aplique a política de cache apropriada.
    - Imagens: para arquivos de imagem, aplique a política de cache apropriada.
5. Aponte o domínio para Azion: atualize suas configurações de DNS para direcionar o registro CNAME do seu domínio para o endpoint Azion fornecido para o seu domínio.
6. Teste e monitore: após configurar a entrega do seu conteúdo, teste-a rigorosamente para garantir que está funcionando conforme esperado e monitore os acessos recebidos.
7. Faça ajustes nas políticas de cache, regras ou outras configurações conforme necessário, com base nas métricas de desempenho e feedback dos usuários.

---

> ## Documentos relacionados

Reviewer: ```suggestion
## Documentação relacionada
```

- [Application Accelerator](https://www.azion.com/en/documentation/products/build/edge-application/application-accelerator/).

> 
> {{end}}

Reviewer: ```suggestion
```
